sirepo/package_data/static/js/sirepo-plotting.js

```diff
@@ -3891,6 +3891,17 @@ SIREPO.app.directive('parameterPlot', function(appState, focusPointService, layo
                     });
             };
 
+            $scope.plotLabels = (plots) => {
```

Comment:
Convention is to omit parens for single parameters: `$scope.plotLabels = plots => {...}`

Reply:
When was this convention made? I don't think this is a good idea. It's always good for automated refactoring/mods to have a consistent syntax. This makes searches for functions easy: `r"\w+\s*=\s*([^\)]+)\s*=>"`

Reply:
I think when we moved to arrow notation a couple of years ago, though I can't find the discussion...

Reply:
> Convention is to omit parens for single parameters

I was responding to this specifically.

Reply:
I mean we made that decision around the same time we moved to arrows. I don't recall strong preference either way though. You make a good point so let's so that way. iow ignore my comment @gurhar1133 :)